src/lib/stores/QTheme.ts

```diff
@@ -45,7 +45,17 @@ interface IMaterialDynamicColorsThemeFormatted {
 type CSSDynamicColor =
   `${keyof IMaterialDynamicColorsThemeColorFormatted}-${keyof IMaterialDynamicColorsThemeFormatted}`;
 
+function extractColorFromCssVar(cssVar: string) {
+  const rootStyles = getComputedStyle(document.documentElement);
+  const varName = cssVar.replace(/var\(([a-z0-9-]+)\)/, "$1");
```

Comment:
Wouldn't it be more efficient to use `cssVar.slice(4, cssVar.length - 1)`? We might want to avoid regex as much as possible